rele/config.py

```diff
@@ -94,7 +94,7 @@ def load_subscriptions_from_paths(sub_module_paths, sub_prefix=None, filter_by=N
             attribute = getattr(sub_module, attr_name)
 
             subscription = subscription_from_attribute(attribute)
-            if not subscription:
+            if not subscription or subscription.topic is None:
```

Comment:
I think this responsibility should fall into `subscription_from_attribute`, which already returns None if a module attribute is not of the right type. This would be a new case, which is, it is a subclass of `Subscription`, but it is "abstract".
That also allows us to log only in that case. Either a warning as @antoniobusrod suggests (thought I wonder whether that is not too aggressive), or an info or even debug (which I think it's more appropriate if abstract subscriptions are documented and valid)